src/t_zset.c

```diff
@@ -1001,22 +1001,20 @@ unsigned char *zzlLastInLexRange(unsigned char *zl, zlexrangespec *range) {
     return NULL;
 }
 
-unsigned char *zzlFind(unsigned char *zl, sds ele, double *score) {
-    unsigned char *eptr = lpSeek(zl,0), *sptr;
+unsigned char *zzlFind(unsigned char *lp, sds ele, double *score) {
+    unsigned char *eptr, *sptr;
 
-    while (eptr != NULL) {
-        sptr = lpNext(zl,eptr);
+    if ((eptr = lpFirst(lp)) == NULL) return NULL;
+    eptr = lpFind(lp, eptr, (unsigned char*)ele, sdslen(ele), 1);
```

Comment:
the improvement is since `lpFind` does the conversion only once, right?
unlike the previous code that used to have it on every call to lpCompare..

Reply:
Yeah, using lpFind will be much faster than using lpCompare directly.